wagtail/wagtailadmin/static_src/wagtailadmin/scss/components/_streamfield.scss

```diff
@@ -6,39 +6,54 @@ ul.sequence {
 li.sequence-member {
     @include clearfix;
     position: relative;
-    border: 1px solid transparent;
+    border: solid $color-grey-4;
     border-width: 1px 0;
 
-    &:hover {
-        background-color: $color-input-focus;
-        border-color: darken($color-input-focus, 10%);
-
-        .sequence-member-inner {
-            > .struct-block > label,
-            > .char_field > label,
-            .sequence > label {
-                opacity: 1;
-            }
-        }
+    &:first-child {
+        border-top: 0;
     }
 
-    &:hover .object-help {
+    // Note this inherits .oject-help from page-editor.scss
+    .object-help {
+        display: block;
+        float: none;
+        font-style: italic;
+        margin: 0;
+        max-width: 650px;
         opacity: 1;
+        padding-left: 0;
+        position: relative;
+        width: 100%;
+
+        &::before {
+            display: none;
+        }
+        @media screen and (max-width: $breakpoint-mobile) {
+            margin: -20px 0 30px;
+            padding-left: 0;
+        }
     }
 
     .struct-block .fields {
-        @include column(10);
+        width: 100%;
         padding-left: 0;
         padding-right: 0;
     }
 
+    .struct-block label {
+        display: block;
+        float: none;
+        text-transform: uppercase;
+        width: 100%;
+    }
+
     .struct-block .sequence-container {
-        @include column(9);
+        width: 100%;
         padding-left: 0;
         padding-right: 0;
 
         .sequence-member-inner {
-            padding: 1.5em 0;
+            padding: 20px 30px 50px 20px;
         }
```

Comment:
I see this is the same value as below – it should be possible to remove the override then.

Reply:
Ah, yes, there was a point at which these had different values when we were thinking of the design. Removed.